### ISSUE TITLE
acdh:hasLiteralIdentifier has two ranges

### ISSUE THREAD

**Issue by zozlak** [acdh-oeaw/arche-schema#28]
The `acdh:hasLiteralIdentifier` has two ranges `xsd:anyURI` and `xsd:string`.

First, it causes trouble for the doorkeeper which does not know what range try to cast this property values to. And it choses wrongly, then e.g. the broken URLs checking script may try to check and report as broken non-URL identifiers.

Second (and more importantly), all URL identifiers should be provided using `acdh:hasIdentifier` so the `acdh:hasLiteralIdentifier` is only meant for values with range `xsd:string`.

Summing up the range of the `acdh:hasLiteralIdentifier` should be set to `xsd:string`

**Comment by zozlak** [comment]
After looking into it a little deeper it turns out we also have `acdh:hasNonLinkedIdentifier` which is actually used far more often (>91k uses on apollo vs 2.7k for the `acdh:hasLiteralIdentifier`).

This suggest we should just drop the `acdh:hasLiteralIdentifier` migrating existing values to `acdh:hasNonLinkedIdentifier`

**Comment by zozlak** [comment]
Thans to @bellerophons-pegasus we realized this is just a common parent for all literal identifier properties and it's not meant to be used directly (see [here](https://github.com/acdh-oeaw/arche-schema/pull/3#issuecomment-692162564)).

We need a convention to distinguish such properties so the doorkeeper can recognize them and forbid their usage. I would propose to leave them without range.

**Comment by zozlak** [comment]
We can also mark them as `acdh:technical`

**Comment by zozlak** [comment]
Turned into #30
